contracts/utils/BridgeMessage.sol

```diff
@@ -166,14 +166,7 @@ library BridgeMessage {
     function decodeEmergencyOpPayload(bytes memory payload) internal pure returns (bool) {
         (uint8 emergencyOpCode) = abi.decode(payload, (uint8));
         require(emergencyOpCode <= 1, "BridgeMessage: Invalid op code");
-
-        if (emergencyOpCode == 0) {
-            return true;
-        } else if (emergencyOpCode == 1) {
-            return false;
-        } else {
-            revert("BridgeMessage: Invalid emergency operation code");
```

Comment:
Do we want to lose this check for the payload value?

Reply:
The else statement cannot be reached because we have `require(emergencyOpCode <= 1, "BridgeMessage: Invalid op code");`